package/src/components/MessageList/MessageList.tsx

```diff
@@ -862,9 +862,7 @@ const MessageListWithContext = <
 
   const goToNewMessages = async () => {
     const isNotLatestSet = channel.state.messages !== channel.state.latestMessages;
-    if (isNotLatestSet && hasNoMoreRecentMessagesToLoad) {
-      loadChannelAroundMessage({});
-    } else if (!hasNoMoreRecentMessagesToLoad) {
+    if (isNotLatestSet) {
```

Comment:
This was just plain wrong at this point, `hasNoMoreRecentMessagesToLoad` should never be `true` when `isNotLatestSet` is also `true` (and yet it is, most likely set somewhere where we don't expect it). I have a guess that there's another bug lying around somewhere but at least the scroll-to behaviour should now work.